src/Components/Bookpage/BookDetails/index.js

```diff
@@ -0,0 +1,19 @@
+function BookDetails({image, title, author, year, pageCount, genre, blurb}) {
+    return ( 
+        <>
```

Comment:
Why do you have fragment element here <>? Wouldn't it be better to wrap in a div the whole component and leave the img loose?